CondFormats/GeometryObjects/src/classes_def.xml

```diff
@@ -95,65 +95,133 @@
    <field name="layerGroupEtaRec" mapping="blob"/>
    <field name="etaTableHF"       mapping="blob"/>
  </class>
- <class name="HcalParameters::LayerItem"/>
+  <class name="HcalParameters::LayerItem"/>
```

Comment:
Only if you need to apply other changes to the code, please also restore the original indentation of this line
```suggestion
 <class name="HcalParameters::LayerItem"/>
```